### PR TITLE
Fixed Issue#25: Create profile page

### PR DESCRIPTION
**Pull request by kartikcode**
Created a Profile Page which would be **specific** to every user for he/she to edit his/her personal details. Implemented upload options for user Profile image.

### CODE REVIEW

**Comment by kartikcode** [comment]
ss1
ss2
ss3
ss4

**Comment by NilayMajorwar** [comment]
Nice work! You'll probably have to change some of the fields though - have a look at the [user model](https://github.com/pclubiitk/Campus-Discuss-Backend/blob/master/users/models.py) in backend for reference (ignore the `activated` and `activation_hash` properties). For now, lets only keep the IITK email id as editable.

Also, I don't see why we need the `className` prop in this component. You can remove that from props, and also remove the `clsx` import then.

File: src/screens/profile/index.js [hunk 1]
```diff
@@ -0,0 +1,342 @@
...
+    name: props.name,
+    userName: props.userName,
+    email: props.mail,
+    fblink: props.fbLink,
+    followers: props.followers,
+    following: props.following,
+    nickname: props.nickname,
+    hall: props.hall,
+  });
+  document.body.style = "background:#f5f5ef ; margin-bottom: 60px;";
```
Comment (NilayMajorwar) [inline src/screens/profile/index.js:65]: Why is this required?

File: src/screens/profile/index.js [hunk 2]
```diff
@@ -0,0 +1,342 @@
...
+    setValues({
+      ...values,
+      [event.target.name]: event.target.value,
+    });
+    count = 0;
+    for (var key in values) {
+      if (values[key]) {
+        count++;
+      }
+    }
```
Comment (NilayMajorwar) [inline src/screens/profile/index.js:78]: A cleaner way for the profile completeness is to keep `count` as a state variable in the component (and rename it to `completenessIndex` or something like that) and here, in this function, set this state variable to the value of `count`.

File: src/screens/profile/index.js (hunk 1 above)
Comment (kartikcode) [reply src/screens/profile/index.js:65]: Oh, I forgot to remove that from props.

Comment (NilayMajorwar) [reply src/screens/profile/index.js:62]: I'm talking about the `document.body.style` line :sweat_smile: . I don't get why we need the bottom margin.
